spanner/crud.js

```diff
@@ -163,6 +163,50 @@ function readData (instanceId, databaseId) {
   // [END read_data]
 }
 
+function readStaleData (instanceId, databaseId) {
+  // [START read_stale_data]
+  // Imports the Google Cloud client library
+  const Spanner = require('@google-cloud/spanner');
+
+  // Instantiates a client
+  const spanner = Spanner();
+
+  // Uncomment these lines to specify the instance and database to use
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Gets a reference to a Cloud Spanner instance and database
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  // Read rows from the Albums table
+  const albumsTable = database.table('Albums');
+
+  const query = {
+    columns: ['SingerId', 'AlbumId', 'AlbumTitle', 'MarketingBudget'],
+    keySet: {
+      all: true
+    }
+  };
+
+  const options = {
+    // Guarantees that all writes that have committed more than 10 seconds ago
+    // are visible
+    exactStaleness: 10
+  };
+
+  albumsTable.read(query, options)
+    .then((results) => {
+      const rows = results[0];
+
+      rows.forEach((row) => {
+        const json = row.toJSON();
+        console.log(`SingerId: ${json.SingerId.value}, AlbumId: ${json.AlbumId.value}, AlbumTitle: ${json.AlbumTitle}, MarketingBudget: ${json.MarketingBudget ? json.MarketingBudget.value : ''}`);
```

Comment:
Can we split these into separate lines? This is a lot to read at once...

Reply:
Done